### PR TITLE
Mc chapter36 bugfixes

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/13930
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/13933

This ticket addresses several bugfixes for the PCPG Chapter 36 form, as outlined in the tickets above. It also adds the standard presubmit information to the form review page. 

## Testing done
- local, all tests pass

## Screenshots
<details><summary>Updated UI</summary>

Screen Shot 2020-09-23 at 3 50 45 PM
Screen Shot 2020-09-23 at 3 51 14 PM
Screen Shot 2020-09-23 at 3 51 26 PM

</details>

## Acceptance criteria
- [x] tests pass
- [x] UI updated

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js [hunk 1]
```diff
@@ -3,63 +3,35 @@ import merge from 'lodash/merge';
...
+        widgetClassNames: 'usa-input-medium',
+      },
     },
   },
   ssn: {
     ...ssnUI,
     'ui:required': () => !hasSession(),
   },
-  vaFileNumber: {
+  VAFileNumber: {
```
Comment (Mottie) [inline src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js:34]: Should this be `VaFileNumber`? I think the backend will have an issue converting camel case to snake case with this naming.

**Review by Mottie — APPROVED** [review]

File: src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js (hunk 1 above)
Comment (micahchiang) [reply src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js:34]: Good question. Not sure. Won't eventual inflection changes on the backend force `VA` for all instances of `vA`, `Va` etc? Still unclear on that whole thing.

Comment (Mottie) [reply src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js:34]: Eventual is an undefined amount of time 😉

Comment (micahchiang) [reply src/applications/vre/28-8832/config/chapters/claimant-information/claimantInformation.js:34]: We took a look at it, and it seems like it won't be an issue when we send the payload to vets-api, so I think we're safe here.

**Pull request merged**